app/models/infra_conversion_job.rb

```diff
@@ -442,13 +440,26 @@ def poll_transform_vm_complete
       raise migration_task.options[:virtv2v_message]
     when 'succeeded'
       update_migration_task_progress(:on_exit)
-      queue_signal(:poll_inventory_refresh_complete)
+      queue_signal(:inventory_refresh)
     end
   rescue StandardError => error
     update_migration_task_progress(:on_error)
     abort_conversion(error.message, 'error')
   end
 
+  def inventory_refresh
+    update_migration_task_progress(:on_entry)
+    if migration_task.options[:destination_vm_uuid].present?
+      target_collection = InventoryRefresh::TargetCollection.new(:manager => migration_task.destination_ems)
+      target_collection.add_target(:association => :vms, :manager_ref => {:ems_ref => migration_task.options[:destination_vm_uuid]})
```

Comment:
I'm not sure if `destination_vm_uuid` is going to work for the `ems_ref` since I don't believe these are the same for ovirt and openstack., you might need a secondary_manager_ref of the vm's uid_ems or you'll need to get the new VM's ems_ref back.